### DISCUSSION TITLE
tailwindcss conflicts with github-markdown-css styles

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#12077] [Help]
tailwindcss conflicts with github-markdown-css styles
, I used prose, but some styles are still affected. Is there any way to isolate this container from the influence of tailwindcss?

**Comment by alex-krasikau** [comment]
In your case, does it make sense not to use `github-markdown.css` file as soon as you use Tailwind Typography to style rich text content?

**Reply by ghost** [reply]
Thanks for the reply, because I use a markdown editor, https://github.com/bytedance/bytemd
When rendering markdown, it will be affected by tailwindcs. I want to find a property to make this markdown container not affected by tailwindcs, similar to making this property use native css.

**Reply by alex-krasikau** [reply]
Can you please better describe the issue? Would be best if you could provide a reproduction.

**Reply by ghost** [reply]
https://github.com/facexyy/test

Thank you very much, this is my example code
It has some differences from native markdown

**Reply by ghost** [reply]
https://bytemd.js.org/playground/

This is official and does not use tailwindcss
